i18n/ja.toml

```diff
@@ -0,0 +1,211 @@
+# site-wide copy
+
+## actions and labels
+[action_get_started]
+other = "始めましょう"
+
+[action_get_started_link]
+other = "/docs/intro/quickstart"
+
+[action_translate]
+other = "このページを翻訳"
+
+[action_translate_link]
+other = "https://github.com/helm/helm-www#internationalization--translation"
+
+[action_copy]
+other = "コピー"
+
+## site-wide banner about versioning
+[banner_1]
+other = "[人種差別は容認できず、Helm プロジェクトの目標と互換性がなく、オープンソースコミュニティに居場所はありません。 #BlackLivesMatter](https://docs.google.com/document/d/1a-lzdtxOlWuzYNGqwlYwxMWADtZ6vJGCpKhtJHHrS54/preview?pru=AAABcqVvUdo*ducxvc88kzBetsUXsohi7A)"
+
+## footer
+[footer_support_title]
+other = "Helm は、400人を超える開発者のコミュニティによってサポートされ、構築されています。"
+
+[footer_support_subtitle]
+other = "...そして他の多くの素晴らしい [helm](https://github.com/helm/helm/blob/master/OWNERS) と [charts](https://github.com/helm/charts/blob/master/OWNERS) コアメンテナー。"
+
+[footer_cncf_status]
+other = "私たちは Cloud Native Computing Foundation の Graduated プロジェクトです。"
+
+# homepage copy
+
+## homepage intro
+[main_title]
+other = "**Kubernetes 用** **パッケージマネージャー**"
+
+[main_subtitle]
+other = "Helm は、[Kubernetes](https://kubernetes.io) 用に構築されたソフトウェアを検索、共有、使用するための最良の方法です。"
+
+## homepage about helm
+[main_what_is_helm_title]
+other = "Helm とは何ですか？"
+
+[main_what_is_helm_description]
+other = """
+Helm は、Kubernetes アプリケーションの管理を支援します。Helm チャートは、最も複雑な Kubernetes アプリケーションの定義、インストール、およびアップグレードを支援します。
+
+チャートは簡単に作成、バージョン管理、共有、公開できます。そのため、Helm の使用を開始し、コピーアンドペーストを停止してください。
+
+Helm は [CNCF](https://cncf.io) の卒業プロジェクトであり、[Helm コミュニティ](https://github.com/helm/community)によって管理されています。
+
+### Learn more:
+
+* [Helm アーキテクチャ](./docs/topics/architecture/)
+* [クイックスタートガイド](./docs/intro/quickstart/)
```

Comment:
These pages don't yet exist in Japanese. That means they are dead links. Can you please start these pages so the linking works.

Reply:
OK